--- common.sh.orig	2023-01-03 15:19:28.698405000 +0800
+++ common.sh	2023-01-03 15:18:32.133374000 +0800
@@ -934,15 +934,9 @@

 # $1 mountpoint to adjust permissions for
 _fix_pot_mountpoint_permissions()
 {
-       local _mp
-       _mp="$1"
-
-       if [ "$(stat -f "%Lp" "${_mp}")" != "700" ]; then
-               _debug "Setting mountpoint permission for $_mp"
-               chmod 700 "$_mp" || exit 1
-       fi
+       true
 }

 # $1 mountpoint to create (proper permissions are applied)
 _create_pot_mountpoint()
