--- start.sh.origin	2022-03-13 17:50:20.253476000 +0800
+++ start.sh	2022-03-13 17:51:46.052399000 +0800
@@ -115,16 +115,20 @@
 }

 _js_create_epair()
 {
-	local _epair
+	local _epair _z
 	_epair=$(ifconfig epair create)
 	if [ -z "${_epair}" ]; then
 		_error "ifconfig epair failed"
 		start-cleanup "$_pname"
 		exit 1 # false
 	fi
-	echo "${_epair%a}"
+
+        _z=${_epair%a}
+        ifconfig ${_z}a ether random
+        ifconfig ${_z}b ether random
+	echo "${_z}"
 }

 # $1 pot name
 # $2 epair interface
